### PR TITLE
Update API

### PR DESCRIPTION
**Pull request by mareklibra**

### CODE REVIEW

File: src/api/swagger.yaml [hunk 1]
```diff
@@ -3262,16 +3274,31 @@ definitions:
       cpu_cores:
         type: integer
         description: Required number of CPU cores
-      ram_gib:
+      ram_mib:
```
Comment (jtomasek) [inline src/api/swagger.yaml:3277]: This most likely requires update in the UI code

File: src/api/types.ts [hunk 2]
```diff
@@ -1105,7 +1168,7 @@ export type HostValidationId =
   | 'container-images-available'
   | 'lso-requirements-satisfied'
   | 'ocs-requirements-satisfied'
-  | 'sufficient-installation-disk-speed'
+  | 'sufficient-or-unknown-installation-disk-speed'
```
Comment (jtomasek) [inline src/api/types.ts:1171]: This key needs to be updated in `config/constants.ts`

**Review by jtomasek — CHANGES_REQUESTED** [review]

File: src/api/swagger.yaml (hunk 1 above)
Comment (mareklibra) [reply src/api/swagger.yaml:3277]: This is new addition to the API, so far not used in the UI.

File: src/api/types.ts (hunk 2 above)
Comment (mareklibra) [reply src/api/types.ts:1171]: fixed

**Review by jtomasek — APPROVED** [review]

**Comment by mareklibra** [comment]
Let's merge this after release today

**Review by jtomasek — APPROVED** [review]

**Pull request merged**